Backend/app/dashboard/CommonAPIs/src/Dashboard/Common/Volunteer.hs

```diff
@@ -0,0 +1,99 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Dashboard.Common.Volunteer
+  ( module Dashboard.Common.Volunteer,
+    module Reexport,
+  )
+where
+
+import Dashboard.Common as Reexport
+import Dashboard.ProviderPlatform.Driver as Reexport (DriverListAPI)
+import Data.Aeson
+import Data.OpenApi hiding (description, name, password, summary, title, url)
+import Data.Text as T
+import Kernel.External.Maps.Types
+import Kernel.Prelude
+import Kernel.Storage.Esqueleto (derivePersistField)
+import Kernel.Types.APISuccess (APISuccess)
+import Kernel.Types.Common
+import Kernel.Types.Id
+import Servant hiding (Summary)
+
+data VolunteerEndpoint
+  = DriverListEndpoint
+  | BookingInfoAPI
+  | AssignCreateAndStartOtpRideAPI
```

Comment:
this datatype required for storing transactions for POST/PUT/DELETE apis, not for GET APIs.
So it should be

```
newtype VolunteerEndpoint
  = AssignCreateAndStartOtpRideEndpoint
```